examples/02_meta-analyses/08_plot_cbma_subtraction_conjunction.py

```diff
@@ -129,7 +131,7 @@
 )
 related_corrected_results = jackknife.transform(related_corrected_results)
```

Comment:
wasn't from this pull request, but I think this variable should have a different name, like `related_diagnostic_results`, so that `related_corrected_results` is not overwritten. follows the convention of going from regular results to corrected results.